### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_View/Select_Link Purpose: "More info" link present in issues table of databases is not descriptive. Screen reader is not announcing link name.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19860]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Benchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG2.4.4; #Link Purpose;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Note:
This issue is observed for all "More info" links present in all issues table

### Repro Steps:

1. Turn on Narrator/NVDA
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment", select few databases and click next.
7. In Assessment results and recommendations step, select a azure sql target and navigate to "View/select" button and press enter
8. After selecting a database in "Assessment results" dialog, issues will be displayed.
9. In issues table, verify whether screen reader is announcing "More info" link descriptively or not

### Actual Result:
"More info" link present in issues table of databases is not descriptive. Screen reader is only announcing as "link More info" but screen reader is not announcing link name i.e Supported compatibility levels in Azure SQL Managed instance.

### Expected Result:
"More info" link present in issues table of databases should be descriptive. Screen reader should announce link name i.e Supported compatibility levels in Azure SQL Managed instance and then it should announce role.

### Attachment:
![NVDA_linkpurpose](https://user-images.githubusercontent.com/83570638/176356957-9570e355-add9-4735-b2de-18824f53aeee.png)
![Link purpose_narrator_](https://user-images.githubusercontent.com/83570638/176357000-9a79dec0-8c26-41a7-bb16-2d4900ef2633.gif)

**Comment by aasimkhan30** [comment]
@brian-harris, can you please add aria-label to these links?

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified the issue in below Environment:
Issue is Fixed
Version: 1.41.1 (user setup)
![Screenshot (26)](https://user-images.githubusercontent.com/100189407/218049102-e44b6d97-c20c-45f4-b932-285f8e05e4ab.png)
![Screenshot (25)](https://user-images.githubusercontent.com/100189407/218049115-1ab10e10-6355-40a8-b588-60309bd3ab6b.png)
